src/main/resources/logstash-rest.yml

```diff
@@ -0,0 +1,38 @@
+# The new settings format:
+# At the top level the label descring the query that will also be used as the
+#   file name for its output.
+#   Below the query label are the following attributes:
+#     * extension - the file extension to be used for output. Currently only .txt and .json are valid. Required
+#     * subdir - some api's are now grouped in a subdirectory of the output directory to lessen clutter. Optional, defaults to root dir.
+#     * retry - whenther if a query fails it will be retried for the configured number of attempts. Optional, defaults to false.
+#     * versions - one or more attributes of the format "version rule: "query string". Each set of version/query key pairs
+#       should evaluate to exactly one that is appropriate for the version of the server being queried. Therefor rules should
+#       be structured in such a way that only a valid query can be executed against a given version. Required.
+#   NPM mode is the only one used: https://github.com/vdurmont/semver4j
+#   NPM Versioning rules are documented here: https://github.com/npm/node-semver
+#
+
+logstash_version:
+  extension: ".json"
+  versions:
+    "> 0.0.0": "/?pretty"
+
+logstash_node:
+  extension: ".json"
+  versions:
+    "> 0.0.0": "/_node?pretty"
```

Comment:
These versions aren't true, right? I forget what version we actually added APIs to Logstash, but I didn't think it was until at least 2.x? Obviously with the `version` check API we can't know until we try.

Reply:
It was 5.x, but if you look, the current version has no checks in code, just instructions not to use is before v.5. So I kept it consistent with what was there for now. The fact that nobody has ever raised an issue on that probably means it never got tried on an earlier version, At some point I'll make another pass through to see if there's any differentiation that requires a version check?